### PR TITLE
On the login screen a warning will pop up if an unsupported/outdated browser is used

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description
We should notify the user if an outdated browser is used

## Screenshots (if appropriate):
bildschirmfoto von 2017-01-18 16-20-46

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: core/templates/login.php [hunk 1]
```diff
@@ -1,10 +1,12 @@
 <?php /** @var $l \OCP\IL10N */ ?>
 <?php
 vendor_script('jsTimezoneDetect/jstz');
+//vendor_script('browser-update/browser-update');
```
Comment (PVince81) [inline core/templates/login.php]: remove?

File: core/templates/login.php [hunk 2]
```diff
@@ -91,4 +93,4 @@
 		</ul>
 	</fieldset>
 </form>
-<?php }
+<?php } ?>
```
Comment (PVince81) [inline core/templates/login.php]: Hmm, I thought we usually never close the php block at the end of files

**Review by PVince81 — COMMENTED** [review]
Great idea. Maybe not annoying enough 😉 

Please remove the commented out line then it's good to go 👍

**Review by pmaier1 — APPROVED** [review]
Fine :+1: Maybe change the text like this if possible (if not, fine as well): "Aktualisieren **S**ie **I**hren Browser..."

**Pull request merged**